src/DotVVM.Framework/Hosting/LocalizablePresenter.cs

```diff
@@ -10,8 +10,10 @@
 
 namespace DotVVM.Framework.Hosting
 {
+    /// <summary>
     /// A DotVVM Presenter that reads culture by <see cref="getCulture" />,
     /// sets the thread culture and invokes the default dotvvm presenter (obtained from IServiceProvider)
+    /// </summary>
     public class LocalizablePresenter : IDotvvmPresenter
```

Comment:
To be honest, I don't like that at all. I don't want to mess the codebase with VisualStudio hacks. The `summary` element is simply totally ugly and it's not required.

* By specification - the specification actually does not force any rules here, the comment just has to be valid XML. There are just recommended tags. (that also means that VS behavior is correct, but in the same way notepad is valid C# IDE)
* By all sane tools I know - Omnisharp and JetBrains rider work.
![image](https://user-images.githubusercontent.com/7894687/34460408-c1ebcc04-ee04-11e7-9af0-380b87c8590e.png)
![image](https://user-images.githubusercontent.com/7894687/34460410-dfc2bf6c-ee04-11e7-85fe-f9fa66f72dca.png)
And if it does not work with any other sane IDE, it can't be a problem to write a plugin or fork and compile :P